Topics/statistics-fundamentals.md

```diff
@@ -0,0 +1,167 @@
+
+### Bayesian Statistics
+
+This last topic is not fundamental for a first pass study, but it's highly recommended since some of Machine Learning algorithms are based on it or leverage some concepts from Bayesian Theory.
+
+The Bayesian approach is often distinguished from the "classical" approach, which you have studied in the [Descriptive Statistics](#Descriptive-Statistics) section, and which is called "frequentist" when compared to the "Bayesian" approach.
+
+The Bayesian approach to statistics is based upon the famous [Bayes Theorem](https://en.wikipedia.org/wiki/Bayes%27_theorem), which states that the probability of an event when prior evidence is given and that might be related to that event.
+
+For example, if cancer is related to age, then, using Bayes’ theorem, a person’s age can be used to more precisely or accurately assess the probability that they have cancer, in contrast to the assessment of the probability of cancer made without knowledge of the person's age.
+
+You can take this course to understand the Bayesian approach to statistics, along with many more concepts, taking the following course:
+
+[**Statistical Rethinking - Video Lectures**](https://www.youtube.com/playlist?list=PLDcUM9US4XdM9_N6XUUFrhghGJ4K25bFc)
+
+You find the complete book from the same author at:
+
+[**Statistical Rethinking - Book**](https://www.amazon.com/Statistical-Rethinking-Bayesian-Examples-Chapman/dp/1482253445)
+
+### Ask Questions
+A rule of thumb rule to learn fast and effectively is to ask questions and read other's questions and answers.
+
+Join communities of people interested in the topic (e.g. Reddit): here you can find discussions, search by keywords (e.g. "standard deviation"), and ask questions, with experts who will answer and help you.
+
+Some _tips_ regarding questions:
+
+- Try to form specific, well-written questions, to minimize the time used by the respondent.
+- Do not ask a question whose answer is found with a quick search on google.
+- If the questions are too general or show laziness they'll likely remain unanswered...
+
+Some subreddits you can subscribe to are:
+
+- [r/Statistics](https://www.reddit.com/r/statistics/)
+- [r/AskStatistics](https://www.reddit.com/r/AskStatistics/)
+- [r/ProbabilityTheory](https://www.reddit.com/r/probabilitytheory/)
+- [r/Bayes](https://www.reddit.com/r/Bayes/)
+
+Two other good places to post (well structured) questions are:
+
+- [HackerNews](http://hn.premii.com/)
+- [Quora](https://www.quora.com/)
+
+### Conclusions
+
+That's all! 
+
+With the above-suggested courses, you should be equipped with most of the statistics knowledge you need to start working on simple Data Science projects. 
+
+The next guide will focus on learning the **Python programming language**, the real tool that will allow you to put into practice the knowledge you've gained with your hard study.
+
+----
+Written by [_clone95_](https://github.com/clone95)
```

Comment:
Another good guide ❤️ please keep them coming!

Reply:
<3 Thanks to your reviews, they are becoming even better :D